app/views/experiences/new.html.erb

```diff
@@ -1 +1,30 @@
+<%= simple_form_for @experience do |f| %>
+<div class="container">
+  <div class="row justify-content-center mt-3">
+    <div class="col-12 col-sm-10 col-md-8 col-lg-6">
+      <div id="experience-desc">
+        <h2 class="mt-2">Let's describe your experience</h2>
+        <p class="mt-5"><strong>What will you and your guests do?</strong></p>
+        <ul>
+          <li>Provide specific plans from start to finish, not multiple ideas or options</li>
+          <li>Describe what makes your experience special—something that guests would not do on their own</li>
+        </ul>
+          <%= f.input :description, as: :text, html5: true, label: false, placeholder: "Tell guests the story they'll do during the experience" %>
+          <a class="btn btn-primary mt-3 smooth" href="#experience-price">Next</a>
+      </div>
+
+
+      <div id="experience-price">
+        <p class="mt-5"><strong>How much is your experience?</strong></p>
+          <%= f.input :price, as: :integer, label: false %>
+          <a class="btn btn-primary mt-3 smooth" href="#">Next</a>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+
+<% end %>
+
```

Comment:
Are you using this file anymore?

Reply:
looks like we don't need it anymore. let's delete it